.github/workflows/tests.yml

```diff
@@ -30,12 +30,14 @@ jobs:
       - name: Fetch flyte code
         uses: actions/checkout@v4
         with:
-          path: "${{ github.workspace }}/flyte"
+          path: flyte
```

Comment:
According to the [docs](https://github.com/actions/checkout), the github workspace is implicit.